mpas_analysis/ocean/streamfunction_moc.py

```diff
@@ -17,8 +16,9 @@
 from ..shared.timekeeping.utility import get_simulation_start_time, \
```

Comment:
Yeah, there's not a clear way to make comments on lines that weren't changed (or near lines that changed).